lib/init-exe/src/main.zig

```diff
@@ -1,6 +1,14 @@
 const std = @import("std");
 
+//pub const log_level: std.log.Level = .info;
+
 pub fn main() anyerror!void {
+    // Note: log messages with info level are by default emitted only in Debug
+    // and ReleaseSafe build modes (and omitted from ReleaseFast and
+    // ReleaseSmall build modes).
+    //
+    // To emit the message in all build modes, either set the root log_level
+    // explicitly or use std.debug.print instead of std.log.info.
     std.log.info("All your codebase are belong to us.", .{});
```

Comment:
Perhaps a simple "Note that info logs are by default only printed in `Debug` and `ReleaseSmall` build modes" suffices? If they want to learn more (maybe they don't) about it, they can simply consult the `std.log` documentation, just like with all other things.

Reply:
>  only printed in Debug and ReleaseSmall build modes"

`Debug` and `ReleaseSafe`.

Reply:
You're right, mixed those up.

Reply:
> Perhaps a simple "Note that info logs are by default only printed in Debug and ReleaseSmallReleaseSafe build modes" suffices? If they want to learn more (maybe they don't) about it, they can simply consult the std.log documentation, just like with all other things.

Thanks @r00ster91! I updated the comment to simplify it.

Cheers,
Robin